Game/Resources/Shaders/Lit/blinn_phong.fs

```diff
@@ -206,9 +214,14 @@ void main()
  	float lit_fragment = ShadowCalculation();
 
 	result += diffuse_color.rgb * ambient * occlusion_color.rgb; //Ambient light
-
+#if ENABLE_LIQUID_PROPERTIES
+    vec3 I = normalize(position - view_pos);
+    vec3 R = reflect(I, fragment_normal);
+	result +=texture(skybox_texture, R).rgb* 0.5;
```

Comment:
you are adding the skybox texture allways without having in consideration if it is ocluded by another object.

Reply:
there should be and option to not have reflections of the skybox at all in my opinion, or at least be able to tweak the value of 0.5